### ISSUE TITLE
Replace batteries in kit for second team

### ISSUE THREAD

**Issue by PeterJCLaw** [srobo/tasks#50]
Context: at kickstart at least one of the team kits in Southampton was used for spares to replace parts which teams has issues with. We need to replace the missing/broken pieces before we can ship the kits to teams.

The remaining kit is going to be sent out as a kit for a school which has two teams. We therefore need to sort that soon, but this isn't blocking any schools getting their (first) kits.

Unfortunately our spare batteries are now all in storage, so fixing this involves a trip to get them.

**Comment by Tyler-Ward** [comment]
This kit needs a battery which are not in southampton

**Comment by Tyler-Ward** [comment]
This kit has been more canabalised for spare parts, however this is not needed for a 2nd team as we have one more than expected, Given how far we are in now i suggest we just use it for more parts and don't try to ship it

**Comment by PeterJCLaw** [comment]
Closing as won't-fix
